src/wp-admin/includes/update-core.php

```diff
@@ -1637,14 +1637,16 @@ function _upgrade_440_force_deactivate_incompatible_plugins() {
 /**
  * @access private
  * @ignore
- * @since 5.9.0
+ * @since 5.8.0 The minimum compatible version of Gutenberg is 10.7.
+ * @since 5.9.0 The minimum compatible version of Gutenberg is 11.9.
+ * @since 6.1.1 The minimum compatible version of Gutenberg is 14.1.
  */
-function _upgrade_590_force_deactivate_incompatible_plugins() {
-	if ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '11.9', '<' ) ) {
+function _upgrade_core_deactivate_incompatible_plugins() {
```

Comment:
@desrosj What do you think?

* Added the @since for each release.
* Renamed the function:
   * Removed the version number.
   * Added `core` to avoid naming conflicts outside of Core.
   * Removed the `force` for readability.
   
BTW Great idea on these changes 👏  You're right. Much better!

Reply:
Looks great!

Only thing that is missing is the original `@since` tag noting when the function was introduced. But much better overall.

Reply:
It was originally `@since 5.8.0` (via c264ba80df1e400d5b1f19fe1a3fda96c7983a54), but the proposed change includes the added GB version detail 👍🏻

Reply:
I removed the comment from `@since 5.8.0` for consistency.

Reply:
@desrosj I'll assume with the final change made and your comment above, that it's okay to commit this PR. I'll mark it for `commit` and do the commit to `trunk`.